ntbs-service/Models/ServiceDirectoryItemWrapper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ntbs_service.Models.Entities;
+using ntbs_service.Models.ReferenceEntities;
+
+namespace ntbs_service.Models;
+
+public class ServiceDirectoryItemWrapper
+{
+    public User User { get; }
+    public PHEC Region { get; }
+    public TBService TBService { get; }
+    public Hospital Hospital { get; }
+
+    public ServiceDirectoryItemWrapper(User user = null, PHEC region = null, TBService tbService = null, Hospital hospital = null)
```

Comment:
Rather than have a single constructor with 4 nullable variables, it would be better to have 4 different constructors, each with a single variable. This will guarantee that the class can only ever be created with one of these properties set.